# diff_drive_controller.yaml
diff_drive_controller:
  ros__parameters:
    type: diff_drive_controller/DiffDriveController
    left_wheel_names: ["left_wheel_joint"]
    right_wheel_names: ["right_wheel_joint"]
    wheel_separation: 0.2
    wheel_radius: 0.033
    publish_rate: 50.0
    base_frame_id: base_link
    use_stamped_vel: false
    command_interfaces: ["velocity"]
    state_interfaces: ["position", "velocity"]
